bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -74,8 +69,46 @@ local_setup() {
     try --max 48 --delay 5 running_nginx
 }
 
+@test 'verify identification errors' {
+    run rdctl snapshot restore 'the-nomadic-pond'
+    assert_failure
+    assert_output --partial $"Error: can't restore snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+    run rdctl snapshot restore 'the-nomadic-pond' --json
+    assert_failure
+    run jq_output '.error'
+    assert_success
+    assert_output --partial $"can't restore snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+    run rdctl snapshot delete 'the-nomadic-pond'
+    assert_failure
+    assert_output --partial $"Error: can't delete snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+    run rdctl snapshot delete 'the-nomadic-pond' --json
+    assert_failure
+    run jq_output '.error'
+    assert_success
+    assert_output $"can't delete snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+}
+
+@test 'can create a snapshot where proposed name is a current ID' {
+    run ls -1 "$PATH_APP_HOME"/snapshots
```

Comment:
Should we have a `PATH_SNAPSHOTS` variable? I'm surprised I don't see any other references to it yet, but won't we need it in the factory-reset tests as well?

Reply:
That's the only occurrence of `"$PATH_APP_HOME"/snapshots` in the code so far.

Testing snapshots can't go in `containers/factory-reset.bats` because of the way it XORs the state of each file before and after a factory-reset (a non-empty snapshots dir breaks that). But sure it makes sense to add that to paths.bash